Dockerfile

```diff
@@ -1,37 +1,26 @@
 FROM ubuntu:18.04
 
-RUN apt-get update
-RUN apt-get -y install curl software-properties-common build-essential
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
-RUN add-apt-repository -y ppa:webupd8team/java
-RUN apt-get update
+RUN apt-get --fix-missing update
+RUN apt-get -y install wget software-properties-common
 
-RUN apt-get -y install nodejs
-RUN echo "oracle-java8-installer shared/accepted-oracle-license-v1-1 select true" | debconf-set-selections
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes --no-install-recommends oracle-java8-installer
+RUN mkdir /opt/adoptopenjdk
+WORKDIR /opt/adoptopenjdk
+
+ENV JDKFILE=OpenJDK11U-jdk_x64_linux_hotspot_11.0.3_7.tar.gz
+ENV JDKNAME=jdk-11.0.3+7
+ENV JDKDIR=jdk-11.0.3+7
+ENV JDKVERSION=11
+RUN wget --quiet https://github.com/AdoptOpenJDK/openjdk$JDKVERSION-binaries/releases/download/$JDKNAME/$JDKFILE
+RUN tar -xf $JDKFILE
+RUN rm $JDKFILE
+ENV PATH="/opt/adoptopenjdk/${JDKDIR}/bin:${PATH}"
 
 RUN mkdir /opt/specmate
 RUN mkdir /opt/specmate/database
 RUN mkdir /opt/specmate/conf
-RUN mkdir /opt/specmate/src
-
-COPY web /opt/specmate/src/web
-COPY bundles /opt/specmate/src/bundles
-
-WORKDIR /opt/specmate/src/web
-RUN LIBSASS_EXT="no" npm run init
-RUN npm run build-prod
 
-WORKDIR /opt/specmate/src/bundles
-RUN sed -i 's/\r$//' ./gradlew
-RUN sed -i 's/^M$//' ./gradlew
-RUN ./gradlew --version
-RUN ./gradlew --no-daemon clean build export -x check
-
-RUN cp specmate-std-env/generated/distributions/executable/dev-specmate-all.jar /opt/specmate/specmate.jar
+COPY bundles/specmate-std-env/generated/distributions/executable/dev-specmate-all.jar /opt/specmate/specmate.jar
```

Comment:
would it make semnse to use the prod version?

Reply:
Would it? I don't have stakes in that. This was rather an arbitrary decision.

Reply:
for the docker version I would use the prod version as it removes some unnecssary bundles